code/crates/core-driver/tests/it/extra.rs

```diff
@@ -1487,7 +1487,7 @@ fn driver_equivocate_vote() {
 }
 
 #[test]
-fn driver_equivocate_proposal() {
+fn driver_equivocating_proposal_others_vote_first() {
```

Comment:
```suggestion
fn driver_equivocating_proposer_others_vote_first() {
```
Similar below. Also can you add some comments summarizing what the new tests do..